cmd/renterd/main.go

```diff
@@ -137,9 +137,9 @@ var (
 			Heartbeat:                      30 * time.Minute,
 			MigrationHealthCutoff:          0.75,
 			RevisionBroadcastInterval:      7 * 24 * time.Hour,
-			ScannerBatchSize:               1000,
-			ScannerInterval:                24 * time.Hour,
-			ScannerNumThreads:              100,
+			ScannerBatchSize:               100,
+			ScannerInterval:                4 * time.Hour,
```

Comment:
any reason for that reduction?

Reply:
I noticed we don't scan a lot of hosts at a time on production. I forgot the actual number but I believe it's less than 20 at a time. I figured we could afford to scan hosts more often. In my head the number was 8 hours, so when I saw it was set to daily I was surprised and that made me lower it.